package.json

```diff
@@ -24,6 +26,7 @@
   },
   "homepage": "https://github.com/lgeiger/node-abi#readme",
   "devDependencies": {
+    "standard": "*",
```

Comment:
I'd prefer using `^8.6.0` instead of `*`.

We have greenkeeper enabled for this repo which will keep everything uptdate.